src/store/reducers/savedAccounts.js

```diff
@@ -62,14 +62,12 @@ const savedAccounts = (state = { accounts: [] }, action) => {
     case actionTypes.removeSavedAccountPassphrase:
       return {
         ...state,
-        accounts: state.accounts.map((account) => {
-          if (!action.data ||
-            (action.data.peerAddress === account.peerAddress &&
-            action.data.passphrase === account.passphrase)) {
+        accounts: [].concat(state.accounts.map((account) => {
```

Comment:
this is removing the peerAddress check, what about customNetwork with different addresses? I think all of their passphrase will be removed.